assignment/assignment-8/assignment-5.html

```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8</title>
+    </head>
+    <body>
+        <h1>Contact Support</h1>
+        <form>
+            <div>
+                <label for="firstNameInput">First Name</label>
+                <input type="text" id="firstNameInput" name="firstName" />
+                <label for="lastNameInput">Last Name</label>
+                <input type="text" id="lastNameInput" name="lastName" />
+            </div>
+
+            <div>
+                <label for="emailInput">Enter your email:</label>
+                <input type="email" id="emailInput" name="email" />
+            </div>
+            <legend>The Priority of the Issue:</legend>
+            <div>
+                <input
+                    type="radio"
+                    id="lowIssueRadio"
+                    name="issue"
+                    value="Low"
+                />
+                <label for="lowIssue">Low</label>
+            </div>
+
+            <div>
+                <input
+                    type="radio"
+                    id="mediumIssueRadio"
+                    name="issue"
+                    value="Low"
+                />
+                <label for="mediumIssueRadio">Medium</label>
+            </div>
+
+            <div>
+                <input
+                    type="radio"
+                    id="highIssueRadio"
+                    name="issue"
+                    value="High"
+                />
+                <label for="highIssueRadio">High</label>
+            </div>
+
+            <div>
+                <h2>The Issue Category</h2>
+                <label>
+                    <input type="radio" id="issueCategory" name="issue" />
```

Comment:
If you are nesting input inside label, no need to use id unless you are using id for CSS or javascript.